mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -66,7 +66,8 @@ class ExhaustiveFeatureSelector(BaseEstimator, MetaEstimatorMixin):
         otherwise.
         No cross-validation if cv is None, False, or 0.
     n_jobs : int (default: 1)
-        The number of CPUs to use for cross validation. -1 means 'all CPUs'.
+        The number of CPUs to use for evaluating different feature subsets
+        in parallel. -1 means 'all CPUs'.
     pre_dispatch : int, or string (default: '2*n_jobs')
         Controls the number of jobs that get dispatched
         during parallel execution in cross_val_score.
```

Comment:
Thanks! Could you also update the pre_dispatch docstring since it still has the cross_val_score referenced there, which may be misleading